src/js/components/Button/__tests__/Button-kind-test.js

```diff
@@ -381,4 +381,53 @@ describe('Button kind', () => {
     fireEvent.mouseOver(getByText('Button'));
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test(`should apply size styling correctly`, () => {
```

Comment:
What is "correctly"?

```suggestion
  test(`should apply styling according to theme size definition`, () => {
```